parcels/tools/exampledata_utils.py

```diff
@@ -79,24 +96,32 @@
 }
 
 
-example_data_url = "http://oceanparcels.org/examples-data"
+def _create_pooch_registry() -> dict[str, None]:
+    """Collapses the mapping of dataset names to filenames into a pooch registry.
 
+    Hashes are set to None for all files.
+    """
+    registry = {}
+    for dataset, filenames in EXAMPLE_DATA_FILES.items():
+        for filename in filenames:
+            registry[f"{dataset}/{filename}"] = None
+    return registry
 
-def get_data_home(data_home=None):
-    """Return a path to the cache directory for example datasets.
 
-    This directory is used by :func:`load_dataset`.
+POOCH_REGISTRY = _create_pooch_registry()
 
-    If the ``data_home`` argument is not provided, it will use a directory
-    specified by the ``PARCELS_EXAMPLE_DATA`` environment variable (if it exists)
-    or otherwise default to an OS-appropriate user cache location.
-    """
+
+def _get_odie(data_home=None):
```

Comment:
What does `odie` stand for? Is it a `pooch`-specific term? Never seen it before...

Reply:
It's a dogs name 😅 . That's the convention that they use in their documentation, calling pooch instances the names of dogs

Reply:
Ah, probably I didn't get the joke because I don't have a dog :-)